index.html

```diff
@@ -145,49 +186,76 @@ <h2 class="title-desk" id="link-1">My recent work</h2>
     <section id="big-version2" class="big-version"></section>
     <section id="big-version3" class="big-version"></section>
     <section class="contact-section" id="contact">
-        <h4>Let's chat!</h4>
-        <p class="contact-text" id="link-3">
-            If you have an application you are interested in developing, a feature that you need built or a project that
-            needs coding. I’d love to help with it
-        </p>
-        <form class="form-container" action="https://formspree.io/f/xdoybyyj" method="post">
-            <label for="user"><input class="form-text" type="text" placeholder="Text Input" name="user-input" required
-                    maxlength="30" id="user" /></label>
-            <label for="email"><input class="form-email" id="email" type="email" placeholder="youremail@email.com"
-                    name="user-email" required /></label>
+      <h4>Let's chat!</h4>
+      <p class="contact-text" id="link-3">
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it
+      </p>
+      
+      <form class="form-container" id="form" action="https://formspree.io/f/xdoybyyj" method="post">
+        <label for="user"><input  class="form-text" type="text" placeholder="Text Input"  name="user-input"  required
+                maxlength="30" id="user" /></label>
+        <label for="email"><input class="form-email" id="email" type="email" placeholder="youremail@email.com"
+                name="user-email" required /></label>
+                <div class="error-message" id="error"></div>
 
-            <textarea class="text-area-color" name="comment" id="user-comment" cols="30" rows="10" maxlength="500"
-                placeholder="Enter text here" required>
-          </textarea>
-            <button class="form-btn" type="submit">Send</button>
-        </form>
+        <textarea class="text-area-color" name="comment" id="user-comment" cols="30" rows="10" maxlength="500" placeholder="Enter text here" required>
+      </textarea>
+        <button class="form-btn" type="submit">Send</button>
```

Comment:
- [ ] Currently, your submit button has a great margin which separates it from the form. Please reduce the margin